tests/linearizability/traffic.go

```diff
@@ -24,7 +24,7 @@ import (
 )
 
 var (
-	DefaultTraffic Traffic = readWriteSingleKey{key: "key", writes: []opChance{{operation: Put, chance: 90}, {operation: Delete, chance: 10}}}
+	DefaultTraffic Traffic = readWriteSingleKey{key: "key", writes: []opChance{{operation: Put, chance: 50}, {operation: Delete, chance: 5}, {operation: Txn, chance: 5}}}
```

Comment:
Shouldn't the three values be summed up to 100?

Reply:
Fixed